src/app/admin-views/accounts/accounts.component.html

```diff
@@ -2,17 +2,44 @@
   <h2 class="title">Account Details</h2>
 
   <div class="card-row">
-    <mat-card routerLink="/admin/accounts/list/hr" class="card">
+
+
+      <mat-card *ngIf="pathParam==='main-company'" routerLink="/admin/accounts/list/company-hrs" class="card">
+        <mat-icon class="icon">person_pin</mat-icon>
+        <div class="title">Company Hrs</div>
```

Comment:
Translation ?

Reply:
this is to verify if the logged in person is a main-company admin or a branch admin